docs/root/intro/version_history.rst

```diff
@@ -18,6 +18,7 @@ Version history
 * build: official released binary is now built against libc++.
 * cluster: added :ref:`aggregate cluster <arch_overview_aggregate_cluster>` that allows load balancing between clusters.
 * config: all category names of internal envoy extensions are prefixed with the 'envoy.' prefix to follow the reverse DNS naming notation.
+* config: use type URL to select an extension whenever the config type URL (or its previous versions) uniquely identify a typed extension.
```

Comment:
This needs to move to the 1.14.0 section

Reply:
Done.